core-web/apps/dotcms-ui/src/app/api/services/dot-gravatar-service.ts

```diff
@@ -27,7 +27,9 @@ interface DotProfile {
  * @export
  * @class DotGravatarService
  */
-@Injectable()
+@Injectable({
+    providedIn: 'root'
```

Comment:
In the modular version of `dot-gravatar-component` this service was provided on `providers` and that module was imported on `dot-toolbar-user` so that component had the injection of this service via modules.
Since now there's no modules, then I had to add `providers` on both components to make it work or add `providers` to the parent component `dot-toolbar`.

I think this service can be a singleton and could be reused in other parts of the project. So I set `providedIn` to `root`.

Also by setting `providedIn` to `root` we can make all the internal components of `dot-toolbar` independent of his parent

Reply:
Also adding services to `providers` on standalone components, creates the needing of `overrideServices` in the tests, because the standalone components does not take the mocks from the `TestBed`